runtime/ops/io.rs

```diff
@@ -50,14 +50,6 @@ static STDOUT_HANDLE: Lazy<StdFile> =
 static STDERR_HANDLE: Lazy<StdFile> =
   Lazy::new(|| unsafe { StdFile::from_raw_fd(2) });
 
-/// Due to portability issues on Windows handle to stdout is created from raw
-/// file descriptor.  The caveat of that approach is fact that when this
-/// handle is dropped underlying file descriptor is closed - that is highly
-/// not desirable in case of stdout.  That's why we store this global handle
-/// that is then cloned when obtaining stdio for process. In turn when
-/// resource table is dropped storing reference to that handle, the handle
-/// itself won't be closed (so Deno.core.print) will still work.
-// TODO(ry) It should be possible to close stdout.
```

Comment:
regarding the TODO is it possible to close stdout after this patch?

Reply:
No, not possible still since they're still held globally here (and not stored in a `Mutex<Option<T>>`).

I removed the todo because I didn't think it was worth keeping it in the code since I think doing this will arise from the need to do it.